### ISSUE TITLE
alias that doesn't exist leads to "ghost node"

### ISSUE THREAD

**Issue by rubo77** [hopglass/hopglass#111]
If you add an alias that suits no noise in the database, you get a link that is always new and online and has the URL

      #!v:m;n:undefined

Example: https://mesh.freifunk.in-kiel.de/#!v:m;n:undefined

**Comment by rubo77** [comment]
I think maybe that "ghost-node" will collect all nodes, that are somehow odd configured, for example don't send an id or such?

**Comment by anoymouserver** [comment]
If you are using the Hopglass-Server the `node_id` is also needed inside the `nodeinfo` part, since the map only gets the data inside the alias objects.
```json
  "deadbeef0202": {
    "nodeinfo": {
      "node_id": "deadbeef0202",
      "hostname": "gw2"
    },
    "firstseen": "2015-09-07T14:53:01"
  },
```

**Comment by rubo77** [comment]
Das löst zumindest den fehler, dass unser vpn2, das gerade offline ist nicht aus der aliases ergänzt wurde, aber der "Seltsame Knoten" ist dadurch ja nicht erklärt.

warum sind einige Knoten angeblich mit dem Knoten, der keine ID hat verbunden?

**Comment by anoymouserver** [comment]
The node with the `undefined` url part is also missing the node_id (as far as I can tell from your [nodes.json](https://hopglass.freifunk.in-kiel.de/nodes.json)).

Is the "Seltsamer unbekannter Knoten" caused by the map server or did you just want to overlay the data for the map?

Nodes that have been updated to 2019.x will continue to be displayed as 'pseudo-online' until they are (auto)removed from the map data, since the map no longer receives any direct data, but they are still listed as neighbors at other nodes which matches the old data.
Possibly the `undefined` node also includes neighbors that have provided invalid mesh data in some way.

**Comment by rubo77** [comment]
Yes, I didn't add a note if, because then it wouldn't be linked to the one that is causing the problem any more

**Comment by rubo77** [comment]
Ich habe herausgefunden, dass die ghost-nodes durch den gluon respondd patch verursacht wurden, dann nur Knoten mit der neuen Firmware Version werden unter "undefined" angezeigt.

siehe https://forum.freifunk.net/t/gluon-2019-1-x-knoten-erscheinen-nicht-in-der-map-respondd/21164
